samples/DrawShapesWithImageSharp/ImageSharpLogo.cs

```diff
@@ -33,23 +40,24 @@ public static void SaveLogo(float size, string path)
                 segments.Add(s);
             }
 
-            var colors = new List<Rgba32>() {
-                Rgba32.FromHex("35a849"),
-                Rgba32.FromHex("fcee21"),
-                Rgba32.FromHex("ed7124"),
-                Rgba32.FromHex("cb202d"),
-                Rgba32.FromHex("5f2c83"),
-                Rgba32.FromHex("085ba7"),
+            var colors = new List<Rgba32>()
```

Comment:
I think this should work:

```suggestion
            var colors = new List<Color>()
```

Reply:
Ah yeah, forgot that.